### DISCUSSION TITLE
I want a media query to detect when a device is running in “night mode”

### DISCUSSION THREAD

**Discussion by Kilian** [WebWeWant/webwewant.fyi#53] [Wants]
With more and more OSes offering “night mode”—where the screen dims and blues are reduced—it would be nice to have a media query to detect that. Maybe even with a way to (request to) deactivate it for a site or a specific part of a website where color accuracy is important.

This would be interesting for media browsing or editing, but also for smaller stuff like adapting the site’s contrast like you might with `light-level` and `prefers-contrast: low` media queries.

<hr>

https://webwewant.fyi/wants/eb00af56654e01082474c2701b4e680c1c729bbf/

**Comment by captainbrosset** [comment]
I believe @martinbalfanz will be interested in this media query request, as he's been looking into new media query implementations in Firefox's style engine.

**Comment by aarongustafson** [comment]
@fantasai Would this be covered by [Dynamic Range in Media Queries 5](https://drafts.csswg.org/mediaqueries-5/#dynamic-range)? Or is it something that has not come up yet?

**Reply by Kilian** [reply]
I don't think so (I researched it for [this article](https://polypane.app/blog/the-complete-guide-to-css-media-queries/#dynamic-range-no-support)), dynamic-range is about detecting "hdr" screens, which doesn't say anything about which colors/what brightness is displaying.
